code/modules/atmospherics/machinery/bluespace_vendor.dm

```diff
@@ -39,12 +41,18 @@
 	var/tank_filling_amount = 0
 	///Base price of the tank
 	var/tank_cost = 10
+	///Has a tank been dispensed from the machine?
+	var/tank_purchased = FALSE
 	///Stores the current price of the gases inside the tank
 	var/gas_price = 0
 	///Helper for mappers, will automatically connect to the sender (ensure to only place one sender per map)
 	var/map_spawned = TRUE
 	///Current operating mode of the vendor
 	var/mode = BS_MODE_OFF
+	///Sound effcts just taken from canisters
```

Comment:
comments shouldn't be like, context for your pr, they should be context for your code (does that make sense?)